lib/clearance/back_door.rb

```diff
@@ -31,8 +31,9 @@ module Clearance
   #   visit new_feedback_path(as: user)
   class BackDoor
     def initialize(app, &block)
-      unless ENV["RAILS_ENV"] == "test"
-        raise "Can't use backdoor outside test environment"
+      @allowed_envs = Clearance.configuration.allowed_envs || ["test", "ci", "development"]
+      unless @allowed_envs.include? ENV["RAILS_ENV"]
```

Comment:
Can we skip the the i-var and use the config directly? Maybe wrap up the check in a well named private method?

Reply:
I agree. It now just checks the variable directly and assumes it's working as expected. Defaults were set in the configuration initializer anyways.